snf-cyclades-app/synnefo/db/models_factory.py

```diff
@@ -16,6 +16,7 @@
 import factory
 from factory.fuzzy import FuzzyChoice
 from synnefo.db import models
+from synnefo.userdata import models as userdata_models
```

Comment:
Since our models live in `synnefo/userdata`, it is preferable to create the factory in that directory. This file is for models that are defined in `synnefo/db/models.py`.